exporter/collector/logs.go

```diff
@@ -27,7 +27,6 @@ import (
 	"cloud.google.com/go/logging"
 	loggingv2 "cloud.google.com/go/logging/apiv2"
 	"google.golang.org/genproto/googleapis/api/monitoredres"
-	monitoredrespb "google.golang.org/genproto/googleapis/api/monitoredres"
```

Comment:
there are more I need to remove, but I want to make sure CI fails without those updates